masterlist.yaml

```
@@ -1804,6 +1804,24 @@ plugins:
         subs: [ '**KFE - Refill Water Bottles.esp**' ]
         condition: 'checksum("KFE - Refill Water Bottles.esp", B85EF60C)'
 
+  - name: 'ROOGNV.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/76452/'
+        name: 'ROOG FNV'
+    req:
+      - *JIPLN
+      - 'YUP - Base Game + All DLC.esm'
+      - 'WeaponRepairKitRepaired.esp'
+      - 'WMIMNV.esp'
```

Master files of a plugin do not need to be added to requirements.
LOOT checks the plugin header and shows an error if any master files are missing.

```suggestion
      - 'WeaponRepairKitRepaired.esp'
```